xAPI.md

```diff
@@ -216,17 +216,22 @@ JSON allows for objects to have properties that contain empty objects. This is n
 * [Actor](#def-actor)
 * [Authentication](#def-authentication)
 * [Authorization](#def-authorization)
-* [Base Endpoint](#def-baseendpoint)
+* [Base Resource Path](#def-baseresourcepath)
 * [Community of Practice](#def-community-of-practice)
-* [Experience API (xAPI)](#def-experience-api)
+* [Experience Application Programming Interface (xAPI)](#def-experience-api)
```

Comment:
I had thought about separating out API as well rather than lumping (you'll see other places where I also lumped rather than created new definitions).  Will define API instead.